core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -144,6 +144,10 @@ otherCommand
     | unset
     | call
     | beginTxn
+    | procStatement
+    ;
+
+procStatement: call | declareCommand | let | executeImmediate | returnStatement
```

Comment:
can we add a catch-all `UnresolvedStatement` for those that we don't parse yet? otherwise it's a regression from `procedureDefinition: string | DBL_DOLLAR`.